libvips/foreign/dzsave.c

```diff
@@ -1999,14 +1999,14 @@ vips_foreign_save_dz_class_init( VipsForeignSaveDzClass *class )
 		_( "Tile width in pixels" ),
 		VIPS_ARGUMENT_OPTIONAL_INPUT | VIPS_ARGUMENT_DEPRECATED,
 		G_STRUCT_OFFSET( VipsForeignSaveDz, tile_size ),
-		1, 8192, 256 );
+		1, 8192, 254 );
 
 	VIPS_ARG_INT( class, "tile_height", 12, 
 		_( "Tile height" ), 
 		_( "Tile height in pixels" ),
 		VIPS_ARGUMENT_OPTIONAL_INPUT | VIPS_ARGUMENT_DEPRECATED,
 		G_STRUCT_OFFSET( VipsForeignSaveDz, tile_size ),
-		1, 8192, 256 );
+		1, 8192, 254 );
```

Comment:
They are just there for compatibility with very old versions of `dzsave`: they add invisible `tile_width` and `tile_height` options which are synonyms for `tile_size`. I think it's correct to change their defaults to match. 

I'm not too worried about the name. It's a deepzoom writer which can optionally make zoomify or maps pyramids as well. It needs to be called `xxxsave` to be consistent with the other operations in that class.

Overlap defaults to 0 for for maps and zoomify, so I think it's pretty safe. You'd have to specifically ask for a zoomify with a 1-pixel overlap, for example, and perhaps someone might want it.